### PR TITLE
Chore: uFuzzy 1.0.6

### PR DESCRIPTION
**Pull request by leeoniya**
Fixes some minor search bugs, plus an additional fix for debounce in Prometheus Metric Encyclopedia

### CODE REVIEW

**Review by bohandley — APPROVED** [review]
LGTM! Thank you @leeoniya 🚀

**Review by konrad147 — APPROVED** [review]
Alerting change looks good

**Review by ashharrison90 — APPROVED** [review]
lgtm 🙌

**Comment by leeoniya** [comment]
thanks, yalls!

File: public/app/plugins/datasource/prometheus/querybuilder/components/MetricEncyclopediaModal.tsx [hunk 1]
```diff
@@ -81,9 +81,24 @@ export const placeholders = {
...
+const uf = new uFuzzy({
+  intraMode: 1,
+  intraIns: 1,
+  intraSub: 1,
+  intraTrn: 1,
+  intraDel: 1,
+});
+
+function fuzzySearch(haystack: string[], query: string, setter: React.Dispatch<React.SetStateAction<number[]>>) {
+  // console.log('fuzzySearch');
```
Comment (joey-grafana) [inline public/app/plugins/datasource/prometheus/querybuilder/components/MetricEncyclopediaModal.tsx:93]: Ooops

Comment (leeoniya) [reply public/app/plugins/datasource/prometheus/querybuilder/components/MetricEncyclopediaModal.tsx:93]: :facepalm: hehe, that was there to help @bohandley verify my debounce fix worked as expected. he's gonna be poking this whole file soon in https://github.com/grafana/grafana/issues/64577, so hopefully he catches it :)

**Comment by leeoniya** [comment]
there's too much diff to backport here. moving to 9.5.0.

@bohandley we should manually backport [the](https://github.com/grafana/grafana/pull/64575/files#diff-3eee20a8edbaf82dbaac97cabbfb6a6823d30905021f2d46aa47a3fe7fb20551) as part of  #64577.

there's a lesson here for me. let's see if i've learned anything for next time :sweat_smile: .

**Pull request merged**